### ISSUE TITLE
Is there a way to let the app remember the location of the user folder?

### ISSUE THREAD

**Issue by choora** [thomasjbradley/todosaurus#20]
Thanks for making this app. It's my favorite todo.txt editor and I installed it as a web app. The only thing I find imperfect is that I have to manually select the location of the local folder every time I start it. Can I set the app to remember the location?

**Comment by thomasjbradley** [comment]
Yeah, it’s a small bug that I still have to fix. Thanks for mentioning it. I’ll get to it as soon as I can.

**Comment by thomasjbradley** [comment]
It’s very exciting to hear that someone else uses the app!
